src/components/auction/OrderbookTable/index.tsx

```diff
@@ -0,0 +1,227 @@
+  }
+  &:first-child {
+    &:before {
+      border-top-left-radius: ${({ theme }) => theme.cards.borderRadius};
+      border-top-right-radius: ${({ theme }) => theme.cards.borderRadius};
+    }
+  }
+  @media (min-width: 768px) {
+    display: flex;
+  }
+  @media (min-width: 1180px) {
+    min-width: auto;
+  }
+`
+
+const StyledEmptyContentWrapper = styled(EmptyContentWrapper)`
+  min-height: 352px;
+`
+
+export const OrderBookTable: React.FC<Props> = () => {
+  const tableData = [
+    {
+      price: '0.07508',
+      amount: '399573787.95950',
+      summary: '2197559000',
+      mySize: '0.00',
+    },
+    {
+      price: '0.07508',
+      amount: '399573787.95950',
+      summary: '2197559000',
+      mySize: '0.00',
+    },
+    {
+      price: '0.07508',
+      amount: '65742129.20000',
+      summary: '2197559000',
+      mySize: '0.00',
+    },
+    {
+      price: '0.07508',
+      amount: '399573787.95950',
+      summary: '2197559000',
+      mySize: '0.00',
+    },
+    {
+      price: '0.07508',
+      amount: '399573787.95950',
+      summary: '2197559000',
+      mySize: '0.00',
+    },
+    {
+      price: '0.07508',
+      amount: '399573787.95950',
+      summary: '2197559000',
+      mySize: '0.00',
+    },
+    {
+      price: '0.07508',
+      amount: '65742129.20000',
+      summary: '2197559000',
+      mySize: '0.00',
+    },
+    {
+      price: '0.07508',
+      amount: '399573787.95950',
+      summary: '2197559000',
+      mySize: '0.00',
+    },
+    {
+      price: '0.07508',
+      amount: '399573787.95950',
+      summary: '2197559000',
+      mySize: '0.00',
+    },
+    {
+      price: '0.07508',
+      amount: '399573787.95950',
+      summary: '2197559000',
+      mySize: '0.00',
+    },
+    {
+      price: '0.07508',
+      amount: '65742129.20000',
+      summary: '2197559000',
+      mySize: '0.00',
+    },
+    {
+      price: '0.07508',
+      amount: '399573787.95950',
+      summary: '2197559000',
+      mySize: '0.00',
+    },
+  ]
+
+  const noAuctions = tableData.length === 0
+
+  return noAuctions ? (
+    <StyledEmptyContentWrapper>
+      <InfoIcon />
+      <EmptyContentText>{noAuctions && 'No auctions.'}</EmptyContentText>
+    </StyledEmptyContentWrapper>
+  ) : (
+    <OverflowWrap>
+      <Table>
+        <StyledRow cols={'1fr 1fr 1fr 1fr'}>
+          <TableCell minWidth={'110px'}>
+            <Wrap>
+              <Wrap margin={'0 10px 0 0'}>Price (DAI)</Wrap>
+              <Tooltip id="info" place={'bottom'} text={`Price (DAI)`} />
+            </Wrap>
+          </TableCell>
+          <TableCell minWidth={'150px'}>
+            <Wrap>
+              <Wrap margin={'0 10px 0 0'}>Amount (GNO)</Wrap>
+              <Tooltip id="info" place={'bottom'} text={`You have no orders for this auction.`} />
+            </Wrap>
+          </TableCell>
+          <TableCell minWidth={'120px'}>
+            <Wrap>
+              <Wrap margin={'0 10px 0 0'}>Summary</Wrap>
```

Comment:
Nit: I think this was also wrong in the design. It should be just "sum" and not "summary". This field will represent the sum of all place orders up to that price

Reply:
done